packages/dcos-integration-test/extra/test_applications.py

```diff
@@ -222,3 +222,23 @@ def assert_packages_match_active_buildinfo(package_ids):
 
         assert set(active_package_ids) <= set(package_ids)
         assert_packages_match_active_buildinfo(active_package_ids)
+
+
+def test_packaging_api(dcos_api_session):
+    """Test the Cosmos API (/package) wrapper
+    """
+    install_response = dcos_api_session.cosmos.install_package('kafka', '1.1.9-0.10.0.0')
```

Comment:
What are the minimum system requirements for launching kafka? We probably want to decorate this test with something that will either skip or fail if the cluster can't possibly support this test.

FYI current PR integration test cluster config is 2 private and 1 public usually. M3.xlarge in provider: aws and m4.xlarge in provider: onprem